static/app/components/events/autofix/autofixMessageBox.tsx

```diff
@@ -466,12 +589,16 @@ function AutofixMessageBox({
                     />
                   )}
                 </AutofixActionSelector>
-              ) : isChangesStep && !prsMade ? (
+              ) : isChangesStep && !prsMade && !branchesMade ? (
                 <AutofixActionSelector
                   options={[
                     {key: 'add_tests', label: t('Add tests')},
                     {key: 'give_feedback', label: t('Iterate')},
-                    {key: 'create_prs', label: t('Approve'), active: true},
+                    {
+                      key: 'create_prs',
+                      label: t('Take it from here'),
```

Comment:
I don't think "Take it from here" as the CTA conveys to me too well what actions are behind the button, but I don't have any better suggestions

Reply:
What about "Use this code"?